### PR TITLE
Feature/dockerize app

### PR DESCRIPTION
**Pull request by paulalden**

### CODE REVIEW

**Review by BertZZ — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2,16 +2,12 @@
...
-_A detailed guide to setting up this Gem is incipient_.
-
-## Installation
-
-Add this line to your application's Gemfile:
+Add the following to your Gemfile:
 
 ```ruby
-gem 'hyrax-hirmeos'
+gem 'hyrax-orcid', git: 'https://github.com/ubiquitypress/hyrax-orcid', branch: 'main'
```
Comment (BertZZ) [inline README.md]: Wait no

**Review by BertZZ — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (paulalden) [reply README.md]: its fine for now. until its moved. you can't install it from rubygems.org anyway (https://rubygems.org/search?query=hyrax-hirmeos) as its not been registered.

File: .circleci/config.yml [hunk 2]
```diff
@@ -55,61 +59,26 @@ jobs:
...
-      - run:
-          command: |
-            if [[ $(command -v cc-test-reporter) == "" ]]; then
-              curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
-              chmod +x ./cc-test-reporter
-            fi
       - run:
-          command: ./cc-test-reporter before-build
+          name: Install Hyrax Orcid
+          command: bundle exec rails g hyrax:orcid:install
```
Comment (BertZZ) [inline .circleci/config.yml]: Why do we need to install Hyrax Orcid?

Comment (paulalden) [reply .circleci/config.yml]: Well spotted. I had left a reference to the other gem i copied this config over from.

Comment (paulalden) [reply .circleci/config.yml]: Fixed now.

File: README.md (hunk 1 above)
Comment (BertZZ) [reply README.md]: Its still the wrong gem name, this is the hirmeos readme

**Review by BertZZ — APPROVED** [review]

**Pull request merged**